contracts/BSCValidatorSet.sol

```diff
@@ -732,7 +737,7 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
       }
 
       // record the jailed validator in validatorSet
-      for (uint k = 0; k < _validatorSet.length; k++) {
+      for (uint k = 0; k < _validatorSet.length; ++k) {
```

Comment:
should we keep the same style? `for (uint k; k < _validatorSet.length; ++k) {`

Reply:
fixed